### PR TITLE
[T4A1][T01-T2]Chua Ping Chan

### PR DESCRIPTION
**Pull request by ChuaPingChan**

### CODE REVIEW

**Comment by ChuaPingChan** [comment]
Still in progress.

**Comment by ChuaPingChan** [comment]
Ready for review.

File: src/seedu/addressbook/data/person/Printable.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	/**
+	 * Returns a printable string of a printable object.
+	 */
+	public abstract String getPrintableString();
+	
+	/**
+	  * Returns a concatenated version of the printable strings of each object.
+	  */
+	public static String getPrintableString(Printable... printables){
+		StringJoiner concatenatedStrings = new StringJoiner(" ");
```
Comment (nishantbudhdev) [inline src/seedu/addressbook/data/person/Printable.java:18]: Avoid implementing methods within the interface.

**Comment by nishantbudhdev** [comment]
@ChuaPingChan Some comments added. Please close the PR after reading them.

**Pull request closed without merging**